src/core/orderbook.js

```diff
@@ -1142,20 +1142,20 @@ OrderBook.prototype.getOffersSync = function() {
 OrderBook.prototype.toJSON =
 OrderBook.prototype.to_json = function() {
   const json = {
-    taker_gets: {
+    gets: {
```

Comment:
Isn't it ambiguous if you don't specify that it is the "taker" as opposed to the "maker"?